src/training/_hooks.py

```diff
@@ -43,9 +43,9 @@ def after_run(self, context, values):
 
 
 class LogSessionHook(tf.train.SessionRunHook):
-  """Logs the session loop by outputting steps, and throughput into logs.
+  """Writes training throughput stats into the logs.
   """
-  _MESSAGE_FORMAT = 'Run: %.2f sec; Steps: %d; Duration: %d sec; Throughput: %.1f instances/sec'
+  _MESSAGE_FORMAT = 'Training throughput: Step %d time: %.2f; Duration: %d sec; Lifetime throughput: %.1f instances/sec'
```

Comment:
Not sure I understand the change here.

Lets keep logs short, so they don't wrap on a normal console width.

1. The first number is not a throughput -- it is time taken to run N steps between successive logs. So it should probably be left at Run.
2. Lifetime is implied.
3. Steps is plural everywhere.